localflavor/np/models.py

```diff
@@ -0,0 +1,91 @@
+"""Nepal specific Model fields"""
+
+import re
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django.core.validators import RegexValidator
+
+from .forms import NPPostalCodeFormField
+from .np_zones import ZONES
+from .np_provinces import PROVINCES
+from .np_districts import DISTRICTS
+
+class NPPostalCodeValidator(RegexValidator):
+    """
+    A validator for Nepali Postal Codes.
+    """
+    default_error_messages = {
+        'invalid': _('Enter a postal code in format XXXXX'),
+    }
+
+    def __init__(self,*args, **kwargs):
+        super().__init__(re.compile(r'^\d{5}$'), *args,**kwargs)
+
+class NPPostalCodeField(models.CharField):
+    """
+        A model field that accepts Nepali postal codes.
+        Format: XXXXX
+        Source: https://en.wikipedia.org/wiki/List_of_postal_codes_in_Nepal
+        .. versionadded:: 4.0
+    """
+    description = _("Postal Code")
+    def __init__(self, *args, **kwargs):
+        kwargs['max_length'] = 5
+        super().__init__(*args, **kwargs)
+        self.validators.append(NPPostalCodeValidator())
+
+    def formfield(self, **kwargs):
+        defaults = {'form_class': NPPostalCodeFormField}
+        defaults.update(kwargs)
+        return super().formfield(**defaults)
+
+class NPDistrictField(models.CharField):
+    """
+        A Modelfield that provides an option to select 
```

Comment:
```suggestion
        A model field that provides an option to select 
```
The same suggestion in other places as well.